apps/chat/src/components/Chat/ChangePathDialog.tsx

```diff
@@ -172,7 +170,7 @@ export const ChangePathDialog = ({
     (parentFolderId = rootFolderId) => {
       const folderName = getNextDefaultName(
         t(DEFAULT_FOLDER_NAME),
-        folders,
+        folders.filter((f) => f.publishedWithMe || f.temporary),
```

Comment:
`ChangePathDialog` can be used not only for `PublishWizard`
```suggestion
        folders.filter((f) => f.folderId === parentFolderId),
```